src/app/component/general/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.html

```diff
@@ -0,0 +1,27 @@
+<div id="list-gallery-content">
+  <div class="list-image">
+    <img class="list-image-content" src="../../../../../../assets/img/tamplate-image-list.png" alt="user added image" />
```

Comment:
If you have more than 2 attributes, please replace it to one column

Reply:
Fixed